src/InvokePlaster.ps1

```diff
@@ -820,14 +812,11 @@ function Invoke-Plaster {
                 # If there is an existing module manifest, load it so we can reuse old values not specified by
                 # template.
                 if (Test-Path -LiteralPath $dstPath) {
-                    $oldModuleManifest = Test-ModuleManifest -Path $dstPath -ErrorAction SilentlyContinue
-                    if ($? -and $oldModuleManifest) {
-                        $props = 'Guid', 'Description', 'DefaultCommandPrefix', 'RootModule', 'AliasesToExport',
-                                 'CmdletsToExport', 'DscResourcesToExport', 'VariablesToExport',
-                                 'FormatsToProcess', 'TypesToProcess', 'ScriptsToProcess', 'NestedModules',
-                                 'PowerShellVersion'
-
-                        CopyModuleManifestPropertyToHashtable $oldModuleManifest $newModuleManifestParams $props
+                    $manifestFileName=Split-Path $dstPath -leaf
```

Comment:
Thanks!  Very minor nit but would you mind putting a space around the `=` on this line.  Ditto for line `818` below.

Reply:
Done.